ocs_ci/utility/aws.py

```diff
@@ -105,6 +105,38 @@ def get_ec2_instance(self, instance_id):
         """
         return self.ec2_resource.Instance(instance_id)
 
+    def get_instances_response_by_name_pattern(self, pattern=None, filter_by_cluster_name=True):
+        """
+        Get the instances by name tag pattern. If not specified it will return all the instances,
```

Comment:
@yitzhak12 , I feel some alignment problem with longer lines in comments. Could you please align it wherever necessary

Reply:
Yes, I will try to avoid long lines